src/main/kotlin/no/nav/fo/forenkletdeploy/service/GitService.kt

```diff
@@ -40,11 +45,25 @@ constructor(
                     ) }
 
     private fun getGithubCommits(application: ApplicationConfig, fromTag: String, toTag: String): List<Commit> =
-            getStashCommits(
-                    fromTag = fromTag,
-                    toTag = toTag,
-                    application = getGithubConfig(application.name)
-            )
+            if (preferGithubChangelog) {
```

Comment:
Ønsker egentlig å bare alltid hente ting fra Githib om det er en github-app. Ønsker å fase ut hele backup mot stash fra github fra FO på sikt. Altså burde applikasjonen hente info om apper i med repo-url som peker mot stash med `StashConsumer` og apper som ligger i Github med `GithubConsumer`.

Altså slippe hele denne if-else-biten her og i tags-delen, og heller ha noe som `if(isGithubRepo)`

Reply:
Enig. Hvis Githubconsumeren funker greit for github-appene deres så fjerner jeg hele if-spetakkelet - det er bare unødvendig kompleksitet.
GithubConsumer inneholder en forenkling ift StashConsumer: ved første gangs deploy vil fromTag være "null". Det er ikke trivielt å finne første commit i github vha rest-api'et. Her har vi ganske enkelt antatt at det fins en committ som er tagget "1" som blir det nye fromTag. Det er veldig mye enklere å legge inn en manuell 1 tag én gang i et prosjekt enn det er å finne første commit (mulig det har blitt enklere med graph-api'et deres, men det har jeg ikke tenkt å bruke tid på)

Reply:
Tenker vi går for å fjerne det, bare sørge for at den bruker riktig consumer. vi må bare teste ordentlig at det fungerer for alle :)